tests/testutils/src/main/kotlin/io/element/android/tests/testutils/EnsureNeverCalled.kt

```diff
@@ -27,3 +27,9 @@ class EnsureNeverCalledWithParam<T> : (T) -> Unit {
         throw AssertionError("Should not be called and is called with $p1")
     }
 }
+
+class EnsureNeverCalledWithTwoParams<T, U> : (T, U) -> Unit {
```

Comment:
About this and the other classes above: maybe we should just use a function?

Something like:

```kotlin
fun ensureNeverCalledWithTwoParams<T, U>(p1: T, p2: U) {
    throw AssertionError("Should not be called and is called with $p1 and $p2")
}
// then use it like
...
onReactionClicked = ::ensureNeverCalledWithTwoParams,
...
```

The syntax looks a bit weirder at the usage point. Feel free to ignore if you think the class approach is cleaner.

Reply:
That's another possible approach, yes. At first I wanted to have parity with `EnsureCalledOnce` and consort.